Makes expected color codes of settrace test results consistent with current void-packages python3-Pygments version.
--- a/test/data/settrace.right
+++ b/test/data/settrace.right
@@ -1,5 +1,5 @@
 (settrace.sh:6):
-6:	  _Dbg_debugger; :
+6:	[37m  [39;49;00m_Dbg_debugger;[37m [39;49;00m:[37m[39;49;00m
 +# Test set_trace call.
 +where
 ->0 in file `settrace.sh' at line 6
